lib/compat/wordpress-6.1/class-wp-theme-json-6-1.php

```diff
@@ -475,6 +470,24 @@ public function get_styles_for_block( $block_metadata ) {
 
 		$current_element = $is_processing_element ? $block_metadata['path'][ count( $block_metadata['path'] ) - 1 ] : null;
 
+		$element_pseudo_whitelist = isset( static::VALID_ELEMENT_PSEUDO_SELECTORS[ $current_element ] ) ? static::VALID_ELEMENT_PSEUDO_SELECTORS[ $current_element ] : array();
```

Comment:
This is the element-specific whitelist of valid pseudo selectors. As things stand only `link` is allowed to have pseudo selectors and these are whitelisted to hover, focus and active.